ethsigner/commandline/src/test/java/tech/pegasys/ethsigner/CommandlineParserTest.java

```diff
@@ -84,39 +89,46 @@ public void fullyPopulatedCommandLineParsesIntoVariables() throws UnknownHostExc
     assertThat(config.getDownstreamHttpRequestTimeout()).isEqualTo(Duration.ofSeconds(10));
     assertThat(config.getHttpListenHost()).isEqualTo("localhost");
     assertThat(config.getHttpListenPort()).isEqualTo(5001);
+    assertThat(config.getTlsOptions()).isNotEmpty();
+    assertThat(config.getTlsOptions().get().getTrustStoreFile())
+        .isEqualTo(new File("./keystore.cert"));
+    assertThat(config.getTlsOptions().get().gettrustStorePasswordFile())
```

Comment:
```suggestion
    assertThat(config.getTlsOptions().get().getTrustStorePasswordFile())
```

Reply:
In vertx doco, its "Truststore" as one word i.e. `getTruststorePasswordFile`